RIOT-OS/RIOT
# tests: remove bogus files

### Contribution description

Two files sneaked into a commit that were not meant to be there. This removes them.

### Testing procedure

Confirm that the files are indeed the output of `make info-modules` and are not needed anymore. I guess I should avoid placing temporary files into the git repo in the future :/

### Issues/PRs references

https://github.com/RIOT-OS/RIOT/pull/18355
